_config.yml

```diff
@@ -1,4 +1,4 @@
 baseurl:    /tal
-pygments: true
+highlighter: pygments
```

Comment:
This change is needed as `pygments: true` has been deprecated, see [this SO](https://github.com/jekyll/jekyll-help/issues/48) for more information.